src/test/emulators/functionsEmulatorRuntime.spec.ts

```diff
@@ -355,7 +356,7 @@ describe("FunctionsEmulator-Runtime", () => {
           request(
             {
               socketPath: runtime.metadata.socketPath,
-              path: "/",
+              path: `/${frb.projectId}/${frb.triggerId}/`,
```

Comment:
Where's the region (here and in all cases)?

Reply:
Are we not supposed to support region-less paths?

Reply:
No I don't think we do/should.